demos/widget/textDir.html

```diff
@@ -0,0 +1,283 @@
+<!doctype html>
```

Comment:
This demo will need to be removed since it's not actually a demo of the widget factory.